### PR TITLE
Release 0.4.0

### PR DESCRIPTION
**Pull request by sethmlarson**
Rebase merge this PR, I'll tag the release commit afterwards to test our deployment.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -7,7 +7,17 @@ name = "truststore"
...
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: MacOS",
+  "Operating System :: Microsoft",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
```
Comment (davisagli) [inline pyproject.toml:19]: Let's wait to claim support for 3.11 until we can successfully test it.

```suggestion
```

**Review by davisagli — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (sethmlarson) [reply pyproject.toml]: Done!

**Comment by sethmlarson** [comment]
@davisagli Looks like the deployment environment is working, it's asking for a review:

Screenshot from 2022-05-05 15-10-42

https://github.com/sethmlarson/truststore/actions/runs/2277996481

I'm going to approve the run.

**Comment by davisagli** [comment]
fwiw I got an email about the pending review of the deployment as well

**Pull request merged**